src/plugin.py

```diff
@@ -93,7 +93,7 @@ def load_plugins(path):
     if path not in sys.path:
         sys.path.insert(0, path)
 
-    ret: dict[str, list] = {'Postgres': [], 'Zookeeper': []}
+    ret: dict[str, list[PostgresPlugin | ZookeeperPlugin]] = {'Postgres': [], 'Zookeeper': []}
```

Comment:
I think it's wrong type: we can't mix PG and ZK plugins in one list. It should be:
`list[PostgresPlugin] | list[ZookeeperPlugin]`